maker.js

```diff
@@ -44,6 +44,7 @@ export async function instantiateMaker(network) {
   }
 
   maker = await Maker.create('browser', config);
+  await maker.authenticate();
```

Comment:
authenticating here lets us fetch on-chain values right away. Of course the user is actually connected right now, so it makes the connect button obsolete. Not sure if that's the UX we're going for but, thought I'd post an example.

Reply:
doesn't this trigger a metamask popup on page load though? (before user interactions)

Reply:
no actually it doesn't because the user doesn't need to sign anything right away. The SDK does add the account right away though. So if you do `maker.listAccounts()` you see the browser account.

Reply:
Wanted to make sure so I just tested it after clearing all my permissions from MM. It doesn't pop up with the connection request on page load. As soon as you click the Connect Button, it does, as expected.